FlightReservation/screens/confirmationScreen.js

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import colors from '../src/colors';
+import TitleFlight from '../components/flightsComponents/titleFlight';
+import BtnNext from '../components/flightsComponents/btnNext';
+import AirplaneIcon from '../components/flightsComponents/airplaneIcon';
+import LocationComponent from '../components/flightsComponents/locationComponent';
+import DateFlight from '../components/flightsComponents/dateFlight';
+import NumPassengers from '../components/flightsComponents/numPassengers';
+
+const ConfirmationScreen = () => {
+  return (
+    <View
+      style={{
+        padding: 30,
+      }}>
+      <View style={[style.rowDirection, style.travelBottom]}>
+        <LocationComponent city="BEG" country="Serbia" side="left" />
+        <AirplaneIcon />
+        <LocationComponent city="AMS" country="Netherlands" side="right" />
+      </View> 
+      <View style={[style.rowDirection, style.dateBottom]}>
+        <DateFlight date={'September 3, 2020'} />
+        <NumPassengers num={3} />
+      </View>
```

Comment:
can you create a reusable component with this? and you can add this on the ConfirmationScreen component and MyFlightInfo

Reply:
Ok, thank you for the advice